### PR TITLE
Improvements to acc code (untested)

### PR DESCRIPTION
**Pull request by Quick-Flash**
All this code was taken from https://github.com/alexeystn/betaflight. It includes improvements to crash handling in angle mode as well as the ability to use silverware acc code. Before we just had taken silverware level code, but now you can choose to enable silverware acc code (untested). Fly with this code and try crashing into objects and see if crash handling is any better, aka the angle after a crash is more accurate with the ground. 

To use silverware acc code in the cli type "set silverware_acc = on"

### CODE REVIEW

**Comment by gretel** [comment]
it's not even compiling.

**Comment by Quick-Flash** [comment]
Fixed it. Forgot to get the new file to compile.

**Comment by gretel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5403347&bid=17712870)

File: src/main/flight/imu_silver.c [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+     float accmag = calcmagnitude(accel);
+
+     if ((accmag > ACC_MIN) && (accmag < ACC_MAX)) {//The bartender makes the fusion if.....
+       // normalize acc
+       for (int axis = 0; axis < 3; axis++) {
+           accel[axis] = accel[axis] / accmag;
+       }
+       // filter accel on to GEstG
+       float filtcoeff = lpfcalc_hz(dt, 1.0f / (float)FILTERTIME);
```
Comment (gretel) [inline src/main/flight/imu_silver.c:97]: Codacy Issue found: [Local variable 'filtcoeff' shadows outer variable](https://app.codacy.com/gh/emuflight/EmuFlight/pullRequest?prid=5403347)

**Comment by Quick-Flash** [comment]
been tested on a whoop. Crash handling and recovery seems to be improved over previous builds, just some better default settings need to be agreed upon.

**Comment by Quick-Flash** [comment]
no good i've got better ideas for this anyhow

**Pull request closed without merging**